docs/configuration.rst

```diff
@@ -63,9 +87,16 @@ To ignore extra cells, you can use the :code:`--nbqa-ignore-cells` CLI argument,
 
     nbqa black my_notebook.ipynb --nbqa-ignore-cells %%html,%%cython
 
-, or the :code:`ignore_cells` option in your :code:`.nbqa.ini` file, e.g.
+, or the :code:`ignore_cells` option in your :code:`setup.cfg` file, e.g.
 
 .. code-block:: cfg
 
     [nbqa.ignore_cells]
     black = %%html,%%cython
+
+, or the :code:`ignore_cells` option in your :code:`pyproject.toml` file, e.g.
+
+.. code-block:: toml
+
+    [tool.nbqa.ignore_cells]
+    black = "%%html,%%cython"
```

Comment:
Wow, awesome!!

Does `toml` require the `%` to be escaped like `cfg` does? (I've opened #256 for the latter, which I forgot about when writing it :flushed: )

Reply:
Thanks for pointing it out. I have no idea on this. Let me read about it and will address this comment accordingly.

Reply:
For TOML, we don't need to escape `%`. I have added a test case to confirm the same.